tck/src/it/resources/application.conf

```diff
@@ -1,40 +1,5 @@
 cloudstate-tck.verify = ["Akka + Node.js", "Akka + Java", "Akka + Scala", "Akka + Go", "Akka + Kotlin"]
 cloudstate-tck.combinations = [{
-  name = "Akka + Node.js"
```

Comment:
@hseeberger You don't need to remove the combinations anymore, you just edit the `verify`-list above, and the TCK will only verify the named combinations.

Reply:
Thanks!